content/02.introduction.md

```diff
@@ -130,6 +147,16 @@ The results of COVID-19 infection can vary greatly among pediatric patients as w
 In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
 Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
 
+Genetic factors may also play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101], including possibly higher risk to individuals of blood group A and possibly lower risk to individuals of type O [@doi:10.1056/NEJMoa2020283].
```

Comment:
The blood group has not been able to replicate in a larger study. I would leave it out.